src/https_client.cc

```diff
@@ -410,6 +410,9 @@ HTTPSResponse HTTPSClient::makeHttpRequest(
         // Request gzip unless downloading files
         poco_req.set("Accept-Encoding", "gzip");
```

Comment:
I've also noticed this suspicious line of code #3780 as it doesn't seem to influence anything.
I did some testing without this line and I did some code history investigation. I've discovered only that it used to be before sending the request and was moved here in the commit 1edcb957eb9db03ce6dd3d0c4abc9e73bef61641. I tested most of the basic operations (create/stop TE, create/change project/client/tag) and also tested sending feedback, everything seems to work.

So my vote is to simply remove this line.

Reply:
I created a separate issue to look into the gzip compression part #3780